File: Lutadores.php
<!DOCTYPE html>
<!--
To change this license header, choose License Headers in Project Properties.
To change this template file, choose Tools | Templates
and open the template in the editor.
-->
<html>
    <head>
        <meta charset="UTF-8">
        <title></title>
    </head>
    <body>
        <?php
        class Lutador{
            private $nome;
            private $idade;
            private $nacionalidade;
            private $vitorias;
            private $derrotas;
            private $empates;
            
            
            
            
            public function lutar(){
            
                
            }

            
            
            
            
            
            
            //Métodos getter e setter
            public function getNome(){
                return $this->nome;
            }
            public function setNome($n){
                $this->nome = $n;
        }
            public function getIdade(){
                return $this->idade;
            }
            public function setIdade($i){
                $this->idade = $i;
        }
            public function getNacionalidade(){
                return $this->nacionalidade;
            }
            public function setNacionalidade($nac){
                $this->nacionalidade = $nac;
        }
            public function getVitorias(){
                return $this->vitorias;
            }
            public function setVitorias($v){
                $this->vitorias = $v;
        }
            public function getDerrotas(){
                return $this->derrotas;
            }
            public function setDerrotas($d){
                $this->derrotas = $d;
        }
            public function getEmpates(){
                return $this->empates;
            }
            public function setEmpates($e){
                $this->empates = $e;
        }
            public function apresentacao($obj){
            echo "Ele vem de {$obj->getNacionalidade()}, tem {$obj->getIdade()} anos, "
            . "vem com uma série de {$obj->getVitorias()} vitórias, {$obj->getDerrotas()} derrotas e {$obj->getEmpates()} empates ."
            . " O nome do lutador éééééé... {$obj->getNome()} !!!!!!";
                
            }
            public function __construct($no, $ida, $na) {
                $this->setNome($no);
                $this->setIdade($ida);
                $this->setNacionalidade($na);
                $this->derrotas = 0;
                $this->empates = 0;
                $this->vitorias = 0;
            }
        }
        
        

        ?>
    </body>
</html>

File: Street fighter JAVA/src/street/fighter/java/Lutadores.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package street.fighter.java;

/**
 *
 * @author Fernando
 */
public class Lutadores {

    private String nome;
    private String idade;
    private String Nacionalidade;
    private int Vitorias;
    private int Derrotas;
    private int Empates;

    public void Lutar() {

    }

    /**
     * @return the nome
     */
    public String getNome() {
        return nome;
    }

    /**
     * @param nome the nome to set
     */
    public void setNome(String nome) {
        this.nome = nome;
    }

    /**
     * @return the idade
     */
    public String getIdade() {
        return idade;
    }

    /**
     * @param idade the idade to set
     */
    public void setIdade(String idade) {
        this.idade = idade;
    }

    /**
     * @return the Nacionalidade
     */
    public String getNacionalidade() {
        return Nacionalidade;
    }

    /**
     * @param Nacionalidade the Nacionalidade to set
     */
    public void setNacionalidade(String Nacionalidade) {
        this.Nacionalidade = Nacionalidade;
    }

    /**
     * @return the Vitorias
     */
    public int getVitorias() {
        return Vitorias;
    }

    /**
     * @param Vitorias the Vitorias to set
     */
    public void setVitorias(int Vitorias) {
        this.Vitorias = Vitorias;
    }

    /**
     * @return the Derrotas
     */
    public int getDerrotas() {
        return Derrotas;
    }

    /**
     * @param Derrotas the Derrotas to set
     */
    public void setDerrotas(int Derrotas) {
        this.Derrotas = Derrotas;
    }

    /**
     * @return the Empates
     */
    public int getEmpates() {
        return Empates;
    }

    /**
     * @param Empates the Empates to set
     */
    public void setEmpates(int Empates) {
        this.Empates = Empates;
    }

}
